SlevomatCodingStandard/Sniffs/ControlStructures/AssignmentInConditionSniff.php

```diff
@@ -54,11 +62,36 @@ public function process(File $phpcsFile, $conditionStartPointer): void
 
 	private function processCondition(File $phpcsFile, int $parenthesisOpener, int $parenthesisCloser, string $conditionType): void
 	{
-		$equalsTokenPointer = TokenHelper::findNext($phpcsFile, T_EQUAL, $parenthesisOpener + 1, $parenthesisCloser);
-		if ($equalsTokenPointer === null) {
+		$equalsTokenPointers = TokenHelper::findNextAll($phpcsFile, T_EQUAL, $parenthesisOpener + 1, $parenthesisCloser);
+		$tokens = $phpcsFile->getTokens();
+
+		if (!$this->ignoreAssignmentsInsideFunctionCalls && count($equalsTokenPointers) !== 0) {
+			$this->error($phpcsFile, $conditionType, reset($equalsTokenPointers));
 			return;
 		}
 
+		foreach ($equalsTokenPointers as $equalsTokenPointer) {
+			$parenthesisStarts = array_keys($tokens[$equalsTokenPointer]['nested_parenthesis'] ?? []);
+			$insideParenthesis = max($parenthesisStarts);
```

Comment:
`max()` does not work with empty array: https://3v4l.org/vhUXS

Reply:
this is true, but we are inside if so nested_parenthesis will be always at least one element, i can add check for empty array, but it will never be hitted by code coverage

Reply:
So ` ?? []` is useless ?

Reply:
Yep, it is for phpstan because nested_parenthesis key is optional

Reply:
It can be ignored in https://github.com/slevomat/coding-standard/blob/master/build/PHPStan/phpstan.neon then.